src/libsyntax/attr.rs

```diff
@@ -77,6 +81,93 @@ pub fn is_used(attr: &Attribute) -> bool {
     })
 }
 
+pub trait AttrMetaListItemMethods {
```

Comment:
Discussed with @nrc over IRC. Conclusion was that we'll keep the two traits since MetaItem and MetaListItems are logically different.